.github/workflows/ci.yaml

```diff
@@ -1,8 +1,9 @@
 name: CI
 on:
   push:
-    branches:
-      - eng
+    branches-ignore:
+      - 'jenkins_sync_*'
+      - 'test_*'
```

Comment:
How about branches like AV-116582-Badges

Reply:
For that branches github actions will run and update their result as per selected branch and It just skip push event for ignore branches

Reply:
lets skip these as well.